contrib/http_base/src/main/java/io/opentelemetry/contrib/http/core/HttpClientHandler.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright 2019, OpenTelemetry Authors
```

Comment:
At this point, this needs to be upped to 2020, on all the new files.

Reply:
Fixed